pkg/config/templates/rancherd-14-loki-stack.yaml

```diff
@@ -0,0 +1,24 @@
+resources:
+  - apiVersion: management.cattle.io/v3
+    kind: ManagedChart
+    metadata:
+      name: loki-stack
+      namespace: fleet-local
+    spec:
+      chart: loki-stack
+      releaseName: loki-stack
+      #      we might want to add this to the cos config eventually, so we can change the version with ldflags
+      version: 2.7.0
+      defaultNamespace: cattle-monitoring-system
```

Comment:
defaultNamespace: cattle-monitoring-system  ??

Reply:
changed to `cattle-logging-system`